src/popup/router/components/BalanceInfo.vue

```diff
@@ -10,7 +10,13 @@
           is-custom
         />
         <span class="display-value text-ellipsis">
-          {{ selectedToken ? selectedToken.convertedBalance : tokenBalance.toFixed(2) }}
+          {{
+            selectedToken
+              ? selectedToken.convertedBalance
+              : typeof tokenBalance === 'object'
+              ? tokenBalance.toFixed(2)
+              : tokenBalance
```

Comment:
This is because `tokenBalance` is a sting before the actual balance arrives, I'm proposing to convert it to BigNumber on loading from `localStorage` or to keep it Number as it was before.
So:
```suggestion
              : tokenBalance.toFixed(2)
```
Or:
```suggestion
              : tokenBalance
```